### DISCUSSION TITLE
[Website Question] Search Results URL?

### DISCUSSION THREAD

**Discussion by Pragma808** [Homebrew/discussions#2197] [Everyday usage] [answered]
Is it possible to view _brew.sh_ search results on a dedicated page instead of the instant lookup from the search box?

I ask because I find myself in the browser more than the terminal, and custom search engines in the address bar is a QoL convenience I really enjoy.

As an example, when I search for Windows packages available through Chocolatey, I am able to enter the search URL like in the image below, and thus search for packages with a query like:   `choco firefox`

![image](https://user-images.githubusercontent.com/59358388/134793838-e6c76241-b6ed-4dd5-9081-03d80395ceb7.png)

Anyhow, I absolutely love brew! Thanks to the developers for this great service!

**Comment by SMillerDev** [comment, marked as answer]
No, the pages are all pre-rendered so there it's no server side component that could handle the search. The current search is fully handled by our sponsor algolia and their api.

**Reply by Pragma808** [reply]
Gotcha. Thank you!
